src/js/ConfigStorage.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// idea here is to abstract around the use of chrome.storage.local as it functions differently from "localStorage" and IndexedDB
+// localStorage deals with strings, not objects, so the objects have been serialized.
+var ConfigStorage = {
+    // key can be one string, or array of strings
+    get: function(key, callback) {
+        if (chrome.storage && chrome.storage.local) {
+            chrome.storage.local.get(key,callback);
+        } else {
+            //console.log('Abstraction.get',key);
+            if(key.forEach) {
```

Comment:
Should this be `Array.isArray(key)`?

Reply:
Yes, maybe is cleaner. And it needs an space between the if and the parenthesis 👾

Reply:
you're probably right on `Array.isArray(key)` I can give it a test.